test/integration/local/payment-request/process-payment-request.test.js

```diff
@@ -94,17 +96,19 @@ describe('process payment requests', () => {
       }]
     })
 
-    expect(invoiceLinesRows[0].schemeCode).toBe('80001')
-    expect(invoiceLinesRows[0].accountCode).toBe('SOS273')
-    expect(invoiceLinesRows[0].fundCode).toBe('DRD10')
-    expect(invoiceLinesRows[0].description).toBe('G00 - Gross value of claim')
-    expect(parseFloat(invoiceLinesRows[0].value)).toBe(25000)
-
     expect(invoiceLinesRows[1].schemeCode).toBe('80001')
     expect(invoiceLinesRows[1].accountCode).toBe('SOS273')
     expect(invoiceLinesRows[1].fundCode).toBe('DRD10')
-    expect(invoiceLinesRows[1].description).toBe('P02 - Over declaration penalty')
-    expect(parseFloat(invoiceLinesRows[1].value)).toBe(-10000)
+    expect(invoiceLinesRows[1].agreementNumber).toBe('SIP00000000000001')
+    expect(invoiceLinesRows[1].description).toBe('G00 - Gross value of claim')
+    expect(parseFloat(invoiceLinesRows[1].value)).toBe(25000)
+
+    expect(invoiceLinesRows[0].schemeCode).toBe('80001')
+    expect(invoiceLinesRows[0].accountCode).toBe('SOS273')
+    expect(invoiceLinesRows[0].fundCode).toBe('DRD10')
+    expect(invoiceLinesRows[0].agreementNumber).toBe('SIP00000000000001')
+    expect(invoiceLinesRows[0].description).toBe('P02 - Over declaration penalty')
+    expect(parseFloat(invoiceLinesRows[0].value)).toBe(-10000)
```

Comment:
Note: For some reason this test failed when I added agreementNumber to the invoiceLine. It was fixed when swapping the rows like this - might be worthwhile checking if there's a deeper reason as to why this happened

Reply:
And by 'added agreementNumber to the invoiceLine', I mean the actual invoice-line.js file

Reply:
Interesting.  I suspect it's because in the `findAll` in the test we're not adding any kind of ordering to the returned invoice lines.

But don't quite understand why the values have been reversed from what they were previously returning.

Odd...

Reply:
Very odd